examples/amp-story/shopping/product.schema.json

```
@@ -0,0 +1,280 @@
+{
+  "$defs": {
+    "https": {
+      "type": "string",
+      "format": "uri-reference",
+      "pattern": "^(/|https://|https?://(127.0.0.1|([^/]+\\.)?localhost)(:[0-9]+)?/)"
+    }
+  },
+  "title": "Product",
+  "description": "A product from an amp story shopping configuration.",
+  "type": "object",
+  "properties": {
```

Should this have `additionalProperties: false`?

```suggestion
  "additionalProperties": false,
  "properties": {
```

---

Yes.